### ISSUE TITLE
JSON error when run command

### ISSUE THREAD

**Issue by brunogusta** [adonisjs/upgrade-kit#5]
### Package version

@adonisjs/upgrade-kit@1.4.0

### Describe the bug

![image](https://github.com/adonisjs/upgrade-kit/assets/50206533/be90bac2-82fb-42f1-8ad3-934fb46fac5a)

I had this bug when try to update the version.

Node version used: v20.9.0

I tried installing the package with pnpm, npm and yarn. With yarn I get a node version error... and the others I get the error above when execute the command.

**Comment by samuel-s-marques** [comment]
Yep. Same here. `SyntaxError: Unexpected token } in JSON at position 497`.

**Comment by samuel-s-marques** [comment]
Hey! I'm trying to run on [thisdatedoesnotexist-server](https://github.com/samuel-s-marques/thisdatedoesnotexist-server). It's still using AdonisJS v5. I'm trying to update it to v6.

**Comment by Julien-R44** [comment]
sorry I missed that issue. i will take a look

**Comment by Julien-R44** [comment]
@samuel-s-marques can you give more information? I just tried to run upgrade-kit on your repo and it seems to works fine. Can you give your node version and also the command/patch you ran?

I'm using Node 21.6

**Comment by royvanderzon** [comment]
I had the same problem and found this code in [https://github.com/adonisjs/upgrade-kit/blob/eecb8e1d56430900f9bf1d0c62647527c5d2635d/src/runner.ts#L41](upgrade-kit/src/runner.ts)

``` javascript
loader.update('Loading config files')
    this.#patchers = this.#options.patchers.map((factory) => factory(this))
    this.rcFile = new RcFile(join(this.#options.projectPath, '.adonisrc.json'))
    this.tsConfigFile = new Tsconfig(join(this.#options.projectPath, 'tsconfig.json'))
    this.pkgJsonFile = new PkgJson(join(this.#options.projectPath, 'package.json'))
```

For me the problem was that in my `tsconfig.json` a line was commeted out:
```
{
    "extends": "adonis-preset-ts/tsconfig.json",
    "include": ["**/*"],
    "exclude": ["node_modules", "build"],
    "compilerOptions": {
        "outDir": "build",
        "rootDir": "./",
        "sourceMap": true,
        "paths": {
            "App/*": ["./app/*"],
            "Config/*": ["./config/*"],
            "Contracts/*": ["./contracts/*"],
            "Database/*": ["./database/*"]
            // "@ioc:/*": ["./contracts/*"]  <------ remove this line
        },
        "types": [
            "@adonisjs/core",
            "@adonisjs/repl",
            "@japa/preset-adonis/build/adonis-typings",
            "@adonisjs/lucid",
            "@adonisjs/auth",
            "adonis5-jwt",
            "@adonisjs/session",
            "@adonisjs/mail",
            "@adonisjs/drive-s3"
        ]
    }
}

```

This was the error for me. I also had another error message but it resolved after I upgraded to node v21.

**Comment by redareda9** [comment]
Same problem here. 

@royvanderzon's solution didn't work. I had no comments in any of my json files.

**Comment by Julien-R44** [comment]
i need a repro or there's nothing I can do to help

**Comment by D40298008** [comment]
I had a similar issue and it wasn't a comment that was causing the problem, but the last element of an object in my `tsconfig.json` had a comma (,)...so it was expecting a new line.  VSCode wasn't complaining so it went unnoticed.
